useful-scripts/importer-tests/index.ts

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable */
+import ZipDirectory from './utils/ZipDirectory';
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { v4 } from 'uuid';
+import fs from 'fs';
+import StreamZip from 'node-stream-zip';
+import globby from 'globby';
+import rimraf from 'rimraf';
+import FormData from 'form-data';
+import Sleep from './utils/Sleep';
+
+// disable insecure warnings
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+dotenv.config();
+const { JWT_TOKEN, API_URL, TOPIC_ID } = process.env;
+
+const createPublication = async () => {
+  console.time('CreatePublication time');
+  try {
+    const res = await axios({
+      method: 'post',
+      url: `${API_URL}/api/publications`,
+      data: {
+        title: `importer-testing-${v4()}`,
+        topicId: TOPIC_ID,
+        contact: {
+          teamName: 'testing',
+          teamEmail: 'johndoe@gmail.com',
+          contactName: 'John Doe',
+          contactTelNo: '123456789',
+        },
+      },
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${JWT_TOKEN}`,
+      },
+    });
+    console.timeEnd('CreatePublication time');
+    console.log(`Created publication. Status code ${res.status}`);
+    const publicationId = res.data.id;
+    await createRelease(publicationId);
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+const createRelease = async (publicationId: string) => {
+  console.time('Create-release-time');
+  try {
+    const res = await axios({
+      method: 'POST',
+      url: `${API_URL}/api/publications/${publicationId}/releases`,
+      data: {
+        timePeriodCoverage: { value: 'AY' },
+        releaseName: 2222,
+        typeId: '1821abb8-68b0-431b-9770-0bea65d02ff0',
+        publicationId: 'c76c1bc4-0a01-4116-74cd-08d8eebbf0c1',
+        templateReleaseId: '',
+      },
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${JWT_TOKEN}`,
+      },
+    });
+    console.timeEnd('Create-release-time');
+    const releaseId = res.data.id;
+    console.log(
+      `Release URL: ${API_URL}/publication/${publicationId}/release/${releaseId}/data`,
+    );
+    await FileOps(releaseId);
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+const FileOps = async (releaseId: string) => {
```

Comment:
I'd rename `FileOps` to `fileOps` so it named consistently with other functions. I'd also consider renaming it to something more meaningful, like `extractZip`.